### DISCUSSION TITLE
Cannot log into https://beta.openlogin.com/

### DISCUSSION THREAD

**Discussion by shanglobus** [Web3Auth/Web3Auth#1169] [Help]
Our staging environment stopped working for everyone all of a sudden. 

Unable to detect login share from the Auth Network. This may be due to slow internet connection. Check your internet speed and try again.

I followed another thread, https://github.com/Web3Auth/web3auth-web/discussions/689

I then go to https://beta.openlogin.com/, I also can't login and got exactly same error. 

<img width="1059" alt="image" src="https://user-images.githubusercontent.com/97058633/210907728-4afa1563-7ddc-4a2e-8050-ea8b750c171d.png">

**Comment by YZhenY** [comment]
Looking into this

**Comment by arsen3d** [comment]
Same problem

**Comment by chaitanyapotti** [comment]
One of our partner endpoints went down for testnet. Services are back up and running now

**Reply by arsen3d** [reply]
Since it's a critical part of infrastructure, can you have its operational status reflected here:
https://status.web3auth.io/
Tech support in the chat window thought there was no problem with the system with this node going down:
https://small-long-brook.ropsten.quiknode.pro/e2fd2eb01412e80623787d1c40094465aa67624a

**Reply by szmizorsz** [reply]
@chaitanyapotti @YZhenY I am facing the same issue on testnet with email and password login through Auth0. However, Twitter login through Auth0 works fine. Both are consistent, the first never let me log in, and the second always succeeds. 
The error is always "Unable to detect login share from the Auth Network.", even if I try it over a high-speed connection.
Could it be caused by the outage of partner endpoints?

**Reply by szmizorsz** [reply]
<img width="1792" alt="Screenshot 2023-01-11 at 14 23 41" src="https://user-images.githubusercontent.com/7248153/211817461-42236f65-9fa5-423c-9d57-ed0609126be9.png">

**Reply by szmizorsz** [reply]
<img width="1792" alt="Screenshot 2023-01-11 at 14 30 18" src="https://user-images.githubusercontent.com/7248153/211818761-05c4e08e-7d7a-4eb6-bbc2-49a21bf048f8.png">

**Reply by szmizorsz** [reply]
My problem was a bad configuration in the login provider configuration, the correct one:

   const web3authProvider = await web3auth.connectTo( WALLET_ADAPTERS.OPENLOGIN, { loginProvider: 'email', extraLoginOptions: { domain: auth0DomainForEmailLogin, verifierIdField: 'email', }, }, );

Instead of 'email' in the 'verifierIdField' there was 'sub'
